tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -64,12 +63,14 @@ spec:
             exit 1
         fi
 
-        prefixedTag=""
+        tagPrefix=$(jq -r '.images.tagPrefix // ""' $DATA_FILE)
         timestampFormat=$(jq -r '.images.timestampFormat // "%s"' $DATA_FILE)
-        if [ $(jq '.images | has("tagPrefix")' $DATA_FILE) == true ]; then
-            prefixedTag="$(jq -r '.images.tagPrefix' $DATA_FILE)-$(date "+$timestampFormat")"
+        timestamp="$(date "+$timestampFormat")"
+        if [ -n $tagPrefix ]; then
+            echo -n "${tagPrefix}-${timestamp}" > $(results.commonTag.path)
+        else
+          echo -n "" > $(results.commonTag.path)
```

Comment:
Indentation differs here from two lines earlier, if that matters.

Reply:
thanks, fixed